include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_merge.h

```diff
@@ -320,8 +335,13 @@ struct __parallel_merge_submitter_large<_IdType, _CustomName,
                         __start = __base_diagonals_sp_global_ptr[__diagonal_idx];
                     }
 
-                    __serial_merge(__rng1, __rng2, __rng3, __start.first, __start.second, __i_elem,
-                                   __nd_range_params.chunk, __n1, __n2, __comp);
+                    auto __ends = __serial_merge(__rng1, __rng2, __rng3, __start.first, __start.second, __i_elem,
```

Comment:
1) const auto
2) we know return type here, why you are using `auto` ?

Reply:
2. cause `auto` is shorter and "faster" to write (a developer doesn't need searching for exact return type for __serial_merge(...) call.

Reply:
As I understand, it doesn't correspondent with recommendations of @akukanov

Reply:
In this context it absolutely doesn't matter.
Even it does, due to that is recommendation, not requirement I would prefer leave it up to a developer.

Reply:
When you add some new parameter, better still to consider the style which already exist in modifying function.